src/auto-evo/simulation/MichePopulation.cs

```diff
@@ -213,15 +213,12 @@ private static void SimulatePatchStep(SimulationConfiguration simulationConfigur
             return;
 
         var leafNodes = new List<Miche>();
-        miche.GetLeafNodes(leafNodes, x => x.Occupant != null);
+        miche.GetLeafNodes(leafNodes, x => x.Occupant is MicrobeSpecies);
 
         // TODO: check if energy should be calculated as doubles because the summed numbers can get pretty high that
         // might benefit from extra precision
         var energyDictionary = species.ToDictionary(x => x, _ => 0.0f);
 
-        // This doesn't use miche.SetupScores as this wants scores for all species, even ones that aren't in the
-        // tree yet
```

Comment:
Why was this comment removed?

Reply:
You removed miche.SetupScores as a function, so I figured we no longer needed to explain why it was not used.

Reply:
Okay that makes sense.